core/templates/pages/learner-dashboard-page/home-tab.component.html

```diff
@@ -1,83 +1,145 @@
-               class="oppia-classroom-link">
-              <div class="classroom-card">
-                <p class="classroom-text">
-                  <span [innerHTML]="'I18N_VIEW_ALL_TOPICS' | translate: {classroomName: item.key}"></span>
-                </p>
+    <div tabindex="0" class="suggested-for-you e2e-test-suggested-for-you" *ngIf="isNonemptyObject(untrackedTopics) && !isGoalLimitReached()">
+      <p class="suggested-for-you-heading" [innerHTML]="'I18N_LEARNER_DASHBOARD_LEARN_SOMETHING_NEW_SECTION' | translate">
+      </p>
+      <div class="suggested-for-you-section">
+        <div *ngFor="let item of untrackedTopics | keyvalue" class="untracked-topics">
+          <div *ngIf="item.value.length !== 0" class="classroom-name">
+            Basic {{item.key}}
+          </div>
+          <div class="topic-summary-tile-scroll">
+            <div class="topic-summary-tile-section d-flex"
+              [ngStyle]="windowIsNarrow && {'width': getWidth(item.value.length) + 'px'}">
+              <div *ngFor="let topicSummaryTile of item.value.slice(0,3)" class="topic-summary-tile">
+                <oppia-learner-topic-summary-tile [topicSummary]="topicSummaryTile">
+                </oppia-learner-topic-summary-tile>
               </div>
-            </a>
+              <a [href]="getClassroomLink(item.key)" target="{{ openInNewWindow ? '_blank' : '_self' }}"
+                 class="oppia-classroom-link">
+                <div class="classroom-card">
+                  <p class="classroom-text">
+                    <span [innerHTML]="'I18N_VIEW_ALL_TOPICS' | translate: {classroomName: item.key}"></span>
+                  </p>
+                </div>
+              </a>
+            </div>
           </div>
         </div>
       </div>
     </div>
-  </div>
-  <div tabindex="0" class="empty-suggested-for-you"
-       *ngIf="isGoalLimitReached()">
-    <p class="suggested-for-you-heading" [innerHTML]="'I18N_LEARNER_DASHBOARD_LEARN_SOMETHING_NEW_SECTION' | translate"></p>
-    <p [innerHTML]="'I18N_LEARNER_DASHBOARD_EMPTY_LEARN_SOMETHING_NEW_SECTION' | translate" class="mt-4"></p>
-    <a class="btn oppia-dashboard-intro-button oppia-transition-200 oppia-learner-dashboard-intro-button-link"
-       [innerHTML]="'I18N_ACTION_BROWSE_LESSONS' | translate"
-       href="/community-library"
-       [routerLink]="communityLibraryUrl"
-       [oppiaFocusOn]="'ourLessonsBtn'">
-    </a>
-  </div>
-  <div tabindex="0" class="empty-suggested-for-you learn-something-new-section"
-       *ngIf="!isNonemptyObject(untrackedTopics) && continueWhereYouLeftOffList.length === 0">
-    <p class="suggested-for-you-heading" [innerHTML]="'I18N_LEARNER_DASHBOARD_LEARN_SOMETHING_NEW_SECTION' | translate">
-    </p>
-    <div class="empty-suggested-for-you-section">
-      <p [innerHTML]="'I18N_LEARNER_DASHBOARD_EMPTY_SUGGESTED_FOR_YOU_SECTION' | translate">
+    <!--[smartRouterLink]="communityLibraryUrl" removed, causes bug-->
```

Comment:
Could you explain why this is commented out i.e. what bug is being referred to?

Reply:
Done. I took a look at develop branch and saw that it was originally routerLink. I'm not sure what happened on my end. Reverting it fixed the issue.